Improve device status page (#1869)
balena-io/docs: pages/learn/manage/device-statuses.md

# Device statuses

The application overview page shows the status of each device in the application.
Device statuses are displayed on the devices page, and the device summary page.  An overview of the devices statuses of an application is shown on the applications page.  Each device can have one of the following statuses:

> <img src="/img/common/main_dashboard/application_device_status.png" alt="Application device status" width="40%" >

Reviewer: @srlowe Checked your PR, are we not using this image for some reason?

# Device statuses

> Device statuses are displayed on the devices page, and the device summary page. An overview of the devices statuses of an application is shown on the applications page. Each device can have one of the following statuses:

Reviewer 2: ```suggestion
Device statuses are displayed on the Devices page and the Device Summary page. An overview of the device statuses of an application is shown on the Applications page. Each device can have one of the following statuses:
```

<img src="/img/common/main_dashboard/application_device_status.png" alt="Application device status" width="40%" >

| Status                     | Description                                                                                                                                                     |
|----------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | **Online**                 | The device is online and is communicating with the cloud within the correct interval. In this state, the device is connected to the VPN, and has recent API communications. |

Reviewer 2: ```suggestion
| **Online**                 | The device is online and is communicating with the cloud within the correct interval. In this state, the device is connected to the VPN and has had recent API communications. |
```

| **Configuring**            | The device is applying OS configuration.                                                                                                                        |
| **Updating**               | The device is updating to a new application release.                                                                                                            |
| **Post Provisioning**      | The device has been [provisioned][device-provisioning] but has not yet come online.                                                                             |
| **Inactive**               | The device has been [deactivated][deactivated] or has been [preregistered][preregistered] but has not yet connected to the {{ $names.cloud.lower }} API.        |
| **Frozen**                 | The device has been frozen because it's outside the organization's allowance, or is in a paid [application type][application type] on a free tier organization. |

## Device Connectivty states

![Device connectivity indicators](/img/common/main_dashboard/device_status.png)

> In addition to the device status, there are indicators to show if the device is having any partial connectivity issue. These indicators can be used to identify common issues, such as a firewall blocking VPN traffic or an inability of the device to communicate with the {{ $names.cloud.lower }} API.

Reviewer 2: ```suggestion
A device's status may indicators of partial connectivity issues on the device, such as a firewall blocking VPN traffic or an inability of the device to communicate with the {{ $names.cloud.lower }} API. These indicators are as follows:
```

> Some statuses may indicate partial connectivity issues:

Reviewer 2: Remove this line if using the suggestion above that ends with "These indicators are as follows:"

Reviewer 2: I'm going to go ahead and force-push this change - looks like it was prematurely resolved

 
<sup>1</sup> `Online (VPN Only)` indicates that the device is unable to communicate with the {{ $names.cloud.lower }} API. 

> <sup>2</sup> `Online (Heartbeat Only)` indicates that device is unable to connect to the {{ $names.cloud.lower }} VPN (e.g. a firewall is blocking VPN traffic) and wouldn't be able to provide remote SSH connections to the device.

Reviewer 2: ```suggestion
<sup>2</sup> `Online (Heartbeat Only)` indicates that device is unable to connect to the {{ $names.cloud.lower }} VPN (e.g. a firewall is blocking VPN traffic) and won't be able to allow remote SSH access to the device.
```

__Note:__ If the device is powered off or loses all network connectivity, it will remain in the `Online (Heartbeat Only)` state until the last target state fetch exceeds the device [API polling interval][poll-interval].

## Debugging Device Status

> If you find your device to be displaying a status which is unclear even with the above notes, visit the [device status][debugging-masterclass#device-status] section in [device debugging masterclass][debugging-masterclass] for more information and what it means for your application.

Reviewer 2: ```suggestion
If you find your device to be displaying a status which is unclear even with the above notes, visit the [device status][debugging-masterclass#device-status] section in the f[device debugging masterclass][debugging-masterclass] for more information.
```
